### DISCUSSION TITLE
callback on update profiles

### DISCUSSION THREAD

**Discussion by reopucino** [gimid/warga#12] [Ideas]
barusan edit profile, tapi saking cepetnya kok ga berasa terjadi apa2 yah? 

Lalu open networktab, dan liat ternyata jalan (walau ga tau berhasil atau gagal)

Nah kayaknya butuh feedback ke usernya soal yang diatas dah.

**Comment by igrir** [comment]
semacam popup atau toast yang ngasih tahu "perubahan berhasil!" begitu ya? mantab sih, boleh boleh

**Reply by reopucino** [reply]
iya kurang lebih begitu.. baik berhasil ataupun gagal, tetep dikasih tau. Jadi usernya ga klik2 tuh update profile ataun spamming tuh button.. hahahahha (Saya sudah melakukannya soalnya)
